packages/nextjs/src/config/webpack.ts

```diff
@@ -62,9 +62,17 @@ export function constructWebpackConfigFunction(
     newConfig.entry = async () => addSentryToEntryProperty(origEntryProperty, buildContext);
 
     // Enable the Sentry plugin (which uploads source maps to Sentry when not in dev) by default
-    const enableWebpackPlugin = buildContext.isServer
-      ? !userNextConfig.sentry?.disableServerWebpackPlugin
-      : !userNextConfig.sentry?.disableClientWebpackPlugin;
+    const enableWebpackPlugin =
+      // TODO: this is a hack to fix https://github.com/getsentry/sentry-cli/issues/1085, which is caused by
+      // https://github.com/getsentry/sentry-cli/issues/915. Once the latter is addressed, this existence check can come
+      // out. (The check is necessary because currently, `@sentry/cli` uses a post-install script to download an
+      // architecture-specific version of the `sentry-cli` binary. If `yarn install`, `npm install`, or `npm ci` are run
+      // with the `--ignore-scripts` option, this will be blocked and the missing binary will cause an error when users
+      // try to build their apps.)
+      ensureCLIBinaryExists() &&
```

Comment:
Should we have this in the webpack plugin? I assume it'll also be a problem for `@sentry/gatsby` (which does similar things)

Reply:
It's a good point. Really we should have it in `sentry-cli` itself. I went ahead and implemented that, but it's currently blocked on unrelated rust errors, so I'm going to merge this in for now. Once those rust problems are fixed (and everyone, including me, is back from winter break), we can switch to that implementation.